src/v2/Apps/Partner/Components/PartnerArtists/PartnerArtistList.tsx

```diff
@@ -0,0 +1,94 @@
+import React from "react"
+import { PartnerArtistList_artists } from "v2/__generated__/PartnerArtistList_artists.graphql"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled, { css } from "styled-components"
+import { Box, Column, GridColumns, Text } from "@artsy/palette"
+import { groupArtists } from "./partnerArtistsUtils"
+import { ColumnSpan } from "@artsy/palette/dist/elements/GridColumns/calculateGridColumn"
+import { Media } from "v2/Utils/Responsive"
+import { PartnerArtistItemFragmentContainer as PartnerArtistItem } from "./PartnerArtistItem"
+import { Carousel } from "v2/Components/Carousel"
+
+export const CssColumns = styled(Box)<{ columnCount }>`
+  ${({ columnCount }) =>
+    css`
+      column-count: ${columnCount};
```

Comment:
Would just set this with a `style={{ columnCount: n }}`

Reply:
Fixed